presto-spi/src/main/java/io/prestosql/spi/eventlistener/SplitCompletedEvent.java

```diff
@@ -99,4 +99,10 @@ public String getPayload()
     {
         return payload;
     }
+
+    @Override
+    public String toString()
+    {
+        return "SplitCompletedEvent[" + queryId + "," + stageId + "," + taskId + "]";
```

Comment:
no space after `,`?

also my IDE offers to generate sth like that
```
@Override
public String toString()
{
    return new StringJoiner(", ", SplitCompletedEvent.class.getSimpleName() + "[", "]")
            .add("queryId='" + queryId + "'")
            .add("stageId='" + stageId + "'")
            .add("taskId='" + taskId + "'")
            .toString();
}
```

Reply:
I can make it this way. A bit more verbse, but probably no harm